### PR TITLE
complete

### PR DESCRIPTION
**Pull request by barsuchellos**
[DEMO LINK](https://barsuchellos.github.io/react_list-of-goods/)

### CODE REVIEW

**Review by Andrew-Skoryk — APPROVED** [review]
LGTM!

File: src/App.tsx [hunk 1]
```diff
@@ -34,62 +36,128 @@ export function getReorderedGoods(
...
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
   // eslint-disable-next-line no-console
   console.log(sortType, isReversed);
```
Comment (Denchepornyuk) [inline src/App.tsx:57]: remove console.log from code

File: src/App.tsx [hunk 2]
```diff
@@ -34,62 +36,128 @@ export function getReorderedGoods(
...
+  sortByName = () => {
+    this.setState({
+      sortType: SortType.ALPHABET,
+    });
+  };
 
-      <ul>
+  sortByLength = () => {
+    this.setState({ sortType: SortType.LENGTH });
+  };
```
Comment (Denchepornyuk) [inline src/App.tsx:88]: you can create one function for that and pass SortType in the argument

File: src/App.tsx [hunk 3]
```diff
@@ -34,62 +36,128 @@ export function getReorderedGoods(
...
-          <li data-cy="Good">Carrot</li>
-          <li data-cy="Good">Eggs</li>
-          <li data-cy="Good">Ice cream</li>
-          <li data-cy="Good">Apple</li>
-          <li data-cy="Good">...</li>
+          {reorderedGoods.map(good => (
+            <li data-cy="Good" key={good}>
+              {good}
+            </li>
+          ))}
```
Comment (Denchepornyuk) [inline src/App.tsx:158]: create a separate component for the goods list

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**